lang/lang_en_sk.txt

```diff
@@ -0,0 +1,1403 @@
+#MSG_IR_03_OR_OLDER c=18
+" 0.3 or older"
+" 0.3 a starsie"
+
+#MSG_FS_V_03_OR_OLDER c=18
+"FS v0.3 or older"
+"FS 0.3 a starsie"
+
+#MSG_IR_04_OR_NEWER c=18
+" 0.4 or newer"
+" 0.4 a novsie"
+
+#MSG_FS_V_04_OR_NEWER c=18
+"FS v0.4 or newer"
+"FS 0.4 a novsie"
+
+#MSG_IR_UNKNOWN c=18
+"unknown state"
+"neznamy stav"
+
+#MSG_MEASURED_OFFSET c=20
+"[0;0] point offset"
+"[0;0] odsadenie bodu"
+
+#MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
+"Crash detection can\x0abe turned on only in\x0aNormal mode"
+"Crash detekcia moze\x0abyt zapnuta len v\x0aNormal mode"
+
+#MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
+"WARNING:\x0aCrash detection\x0adisabled in\x0aStealth mode"
+"POZOR:\x0aCrash detekcia\x0adeaktivovana v\x0aStealth mode"
+
+#MSG_BABYSTEPPING_Z c=15
+"Adjusting Z:"
+"Doladenie Z:"
+
+#MSG_SELFTEST_CHECK_ALLCORRECT c=20
+"All correct"
+"Vsetko OK"
+
+#MSG_WIZARD_DONE c=20 r=3
+"All is done. Happy printing!"
+"Vsetko je hotove!"
+
+#MSG_AMBIENT c=14
+"Ambient"
+"Okolie"
+
+#MSG_AUTO c=6
+"Auto"
+"\x00"
+
+#MSG_PRESS c=20 r=2
+"and press the knob"
+"a stlacte tlacidlo"
+
+#MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
+"Are left and right Z~carriages all up?"
+"Dosli oba Z voziky k~hornemu dorazu?"
+
+#MSG_AUTO_HOME c=18
+"Auto home"
+"\x00"
+
+#MSG_AUTOLOAD_FILAMENT c=18
+"AutoLoad filament"
+"Autozav. filam."
```

Comment:
You don't need to shorten `filam.` as `Autozav. filament` will fit in the 18 chars limit.
Or you may be able to `Autozaviest fil.` instead.